full/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM buildpack-deps:disco
+FROM buildpack-deps:bionic
```

Comment:
Yeah, I am downgrading ubuntu because disco is EOL

Reply:
@RDIL No need, see https://github.com/gitpod-io/workspace-images/blob/master/full/Dockerfile#L1